packages/components/psammead-social-embed/src/CaptionWrapper/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { node, string } from 'prop-types';
+import styled from 'styled-components';
+import { C_WHITE } from '@bbc/psammead-styles/colours';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { GEL_BREVIER } from '@bbc/gel-foundations/typography';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { visuallyHiddenStyle } from '../utilities';
+
+const C_BLACK = '#000000';
+
+const Figure = styled.figure`
+  background-color: ${C_BLACK};
```

Comment:
I wonder if this styling is correct, it's not used on the new article/story pages, captions are on a white background, we should check with UX

Reply:
It's `#000000` in the design. There'll be a chance for them to review it before it goes live. 👍